Library/Formula/utf8proc.rb

```diff
@@ -1,20 +1,16 @@
 class Utf8proc < Formula
   desc "Clean C library for processing UTF-8 Unicode data"
   homepage "https://juliastrings.github.io/utf8proc/"
-  url "https://github.com/JuliaStrings/utf8proc/archive/v2.8.0.tar.gz"
-  sha256 "a0a60a79fe6f6d54e7d411facbfcc867a6e198608f2cd992490e46f04b1bcecc"
+  url "https://github.com/JuliaStrings/utf8proc/releases/download/v2.9.0/utf8proc-2.9.0.tar.gz"
+  sha256 "bd215d04313b5bc42c1abedbcb0a6574667e31acee1085543a232204e36384c4"
   license all_of: ["MIT", "Unicode-DFS-2015"]
 
   bottle do
-    sha256 "e39f7d87cb5ba31a4f535ac6d01ebfb66e7fe99cc35a99b5398ce584e780c3f9" => :tiger_altivec
   end
 
-  # Unbreak build on legacy compilers which lack warnings for sign conversion
+  # Unbreak build on legacy compilers which lack warnings for sign conversion & linking
   patch :p0, :DATA
 
-  # Uses -compatibility_version, not present in Tiger
```

Comment:
It's still using `-compatibility_version` and passing that straight to `ld`, which is failing for me on Tiger; I think this does still need ld64.

Reply:
hmm, interesting, I've managed to recreate the issue. I wasn't hitting it because I built it using GCC 4.0, but if I install apple-gcc42, and use that, I hit the issue you mention. Thing is though regardless of either compiler, the same flags are used to build. I looked at the command output and `-Wl,-compatibility_version -Wl,3 -Wl,-current_version -Wl,3.0.0` is passed both times, it just happens to succeed when it's GCC 4.0.
I'll try and dig further to understand why it works on GCC 4.0, my guess is that the error is there but 4.0 just silently ignores it. I'll follow up over the next few days :)

Reply:
I typed a reply and pressed resolve conversion, should've pressed comment first.
Found the issue, apple-gcc42 is stricter than 4.0 on Tiger and treats those `-Wl` flags as linker flags where as the build is meant to be using libtool as implied by `-dynamiclib` and `-compatibility_version` is a libtool flag on Tiger. Dropping the `-Wl` flags resolves the issue.
Tested with GCC 4.0 & 4.2 on Tiger ppc, 10.5, 10.6, 10.11 on intel with their default compilers.

Reply:
> treats those -Wl flags as linker flags 

Interestingly the manpage for 4.0 still says that `-Wl` passes things to the linker, but I guess there are edge cases that got cleaned up?

Reply:
Oh, interesting. I have found that 4.0 is definitely much more lenient, there is another scenario in the back of my mind which I can't recall at the moment where there is an issue and 4.0 ploughs through.

Reply:
The other thing is that by leopard, `ld` might have supported `-compatibility_version` so GCC 4.2 passing to `ld` would have the desired result.